web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
+ - **The client process** - A React app that implements your app's frontend.
+
+    During development, this is a dev server with hot reloading. In production,
+    it's a Node process that serves pre-built static files with environment variables
+    embedded during the build.
+
+ - **The server process** - An Express server that implements your app's backend.
+ 
+  During development, this is an Express server controlled by a
+  [`nodemon`](https://www.npmjs.com/package/nodemon) process that takes care of
+  hot reloading and restarts. In production, it's a regular Express server run
+  using Node.
+
+Check [the introduction](/docs) for a more in-depth explanation of Wasp's runtime architecture.
+
+You can configure both processes through environment variables. See [the
+deployment instructions](/docs/advanced/deployment/manually#environment-variables) for a full list
+of supported variables.
+
+Wasp gives you runtime access to the processes' configurations through **configuration objects**.
+
+## Server configuration object
+
+The server configuration object contains these fields:
+
+- `frontendUrl: String` - Set it with env var `WASP_WEB_CLIENT_URL`.
```

Comment:
I've only documented the stuff we mention in other parts of our docs, which is only `frontendUrl`.

We should discuss which env variables we want to expose and can then make an issue for adding that to the docs.